src/components/ProjectsCard/Project.styled.tsx

```diff
@@ -0,0 +1,73 @@
+import {

```

Comment:
- change the file name `cards.styled.ts`
- no need to `style.ts` delete it please!

Reply:
Ok, done